definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -12346,5 +12346,61 @@
     "CaseFieldID": "ApplyForDecreeAbsolute",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DateRespondentEligibleForDA",
+    "UserRole": "citizen",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DateRespondentEligibleForDA",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "25/04/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "DateRespondentEligibleForDA",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
```

Comment:
Dunno if superuser usually have greater than R permission

Reply:
Yup, I'd advise CRUD for superuser in all the fields

Reply:
ok. I'll give it CRUD.
But shouldn't the superuser only be used for changing states?